elao.infra/.manala/docker/Dockerfile.terraform.tmpl

```diff
@@ -1,21 +1,37 @@
 FROM golang:alpine3.9 AS provider-openstack
```

Comment:
Fyi, base image such as `golang:alpine3.9` could be defined as an `ARG` to avoid repetition :)
(let's keep that for alpine 3.11 upgrade)

Reply:
nice to know 👍